### PR TITLE
qe: add QE on Tectonic installation guides

### PR DESCRIPTION
**Pull request by robszumski**
Moving the QE on Tectonic docs from their old location to here, in order to keep them all in the same place.

### CODE REVIEW

**Comment by robszumski** [comment]
cc @josephschorr

**Comment by josephschorr** [comment]
LGTM

File: quay-enterprise/tectonic/index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+[quay-enterprise-tour]: https://quay.io/tour/enterprise
+
+## Prerequisites
+
+A PostgreSQL database must be available for Quay Enterprise metadata storage.
+We currently recommend running this database server outside of the cluster.
+
+## Download Kubernetes Configuration Files
+
+Visit your [Tectonic Account][tectonic-account] and download the pre-formatted pull secret, under "Account Assets". There are several formats of the secret, be sure to download the Kubernetes-formatted file. This pull secret is used to download the Quay Enterprise containers.
```
Comment (chancez) [inline quay-enterprise/tectonic/index.md]: Visit your [Tectonic Account][tectonic-account] and download the pre-formatted pull secret, under "Account Assets". There are several formats of the secret, be sure to download the "dockercfg" format resulting in a `config.json` file. This pull secret is used to download the Quay Enterprise containers.

File: quay-enterprise/tectonic/index.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+## Deploy to Kubernetes
+
+All Kubernetes objects will be deployed under the "quay-enterprise" namespace.
+The first step is to create this namespace:
+
+```sh
+kubectl create -f quay-enterprise-namespace.yml
+```
+
+Next, add your pull secret to Kubernetes:
```
Comment (chancez) [inline quay-enterprise/tectonic/index.md]: Next, add your pull secret to Kubernetes (make sure you specify the correct path to `config.json`):

File: quay-enterprise/tectonic/index.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+```sh
+kubectl create -f coreos-pull-secret.yml --namespace=quay-enterprise
```
Comment (chancez) [inline quay-enterprise/tectonic/index.md]: kubectl create secret generic coreos-pull-secret --from-file=".dockerconfigjson=config.json" --type='kubernetes.io/dockerconfigjson' --namespace=quay-enterprise

**Comment by robszumski** [comment]
@chancez is gonna make these changes after merge

**Pull request merged**